app/client/components/helpCentre/helpCentreSeeAllTopics.tsx

```diff
@@ -0,0 +1,113 @@
+import { css } from "@emotion/core";
+import { space } from "@guardian/src-foundations";
+import { brand, neutral } from "@guardian/src-foundations/palette";
+import { headline } from "@guardian/src-foundations/typography";
+import { RouteComponentProps } from "@reach/router";
+import React from "react";
+import { trackEvent } from "../analytics";
+import { Button } from "../buttons";
+import { SectionHeader } from "../sectionHeader";
+import { SectionPageContainer } from "../sectionPageContainer";
+import { helpCentreConfig, helpCentreNavConfig } from "./helpCentreConfig";
+import { HelpCentreMoreTopics } from "./helpCentreMoreTopics";
+import {
+  linkAnchorStyle,
+  linkArrowStyle,
+  linkListItemStyle,
+  linksListStyle
+} from "./helpCentreStyles";
+
+interface HelpCentreProps extends RouteComponentProps {
+  topicCode?: string;
+}
+
+const h2Css = css`
+  margin: 0 ${space[3]}px ${space[9]}px ${space[3]}px;
+  border-top: 1px solid ${neutral["86"]};
+  ${headline.small({ fontWeight: "bold" })}
+`;
+
+const liCss = (questionIndex: number) => css`
+  ${linkListItemStyle};
+  border-top: ${questionIndex === 0 ? "1px solid #DCDCDC" : "none"};
+`;
+
+const divCss = css`
+  margin: 67px ${space[3]}px 0 ${space[3]}px;
+`;
+
+const buttonDivCss = css`
+  margin-top: ${space[4]}px;
+  padding-top: ${space[4]}px;
+  border-top: 1px solid ${neutral["86"]};
+`;
+
+const HelpCentreSeeAll = (props: HelpCentreProps) => {
+  const selectedTopicObject = helpCentreNavConfig.find(
+    topic => topic.id === props.topicCode
+  );
+
+  const helpCentreConfigObject = helpCentreConfig.find(
+    topic => topic.id === selectedTopicObject?.id
+  );
```

Comment:
Agreed `selectedNavTopic` or `selectedNavTopicObject` is a better name for line 46 but on line 50 it's a bit trickier to name. If line 50 is changed to `selectedTopic` on line 93 there'd be `!selectedTopic` which wouldn't be right. 

Though this will all change when things are transferred over to s3 as it's down to the way the `helpCentreConfig.ts` file is currently organised. I'm not sure what name would best communicate what line 50-52 is attempting to do...